lib/gcloud/pubsub/project.rb

```diff
@@ -125,15 +92,22 @@ def get_topic topic_name
       #   An optional Hash for controlling additional behavior. (+Hash+)
       # <code>options[:autocreate]</code>::
       #   Flag to control whether the topic should be created when needed.
```

Comment:
"the topic should be created when needed" -> "the requested topic should be created if it does not exist"